iam/v1/service.proto

```diff
@@ -62,6 +62,10 @@ message ContactInfoValue {
   org.packetbroker.v3.ContactInfo value = 1;
 }
 
+message TargetValue {
+  org.packetbroker.v3.Target value = 1;
+}
```

Comment:
Why is this necessary? `org.packetbroker.v3.Target` is already a message, so you can already tell the difference between unset and zero value.

Reply:
It's used in update only, to indicate that the value should be taken.

Now, if `target` is unset, there's no update of target. Note that we don't use field masks in this API. If `target.value` is unset, clear the target information.

Reply:
Can you write that in a comment so that it's clear to users (and our future selves)?